pkg/channel/http/http_channel.go

```diff
@@ -50,6 +50,9 @@ const (
 
 // Channel is an HTTP implementation of an AppChannel.
 type Channel struct {
+	// TODO: Remove flag once feature is ratified
+	NoDefaultContentType bool
```

Comment:
nit: why upper case?

Reply:
Needs to be exported because of https://github.com/dapr/dapr/pull/4491/files#diff-e56f9bb947923c6f8bb4c995bf4b9469a0ed117276223d16b177dcb04399a4bcR2234